Extension/package.json

```diff
@@ -1588,6 +2050,7 @@
     "@typescript-eslint/parser": "^2.19.2",
     "async-child-process": "^1.1.1",
     "await-notify": "^1.0.1",
+    "editorconfig": "^0.15.3",
```

Comment:
Will you be checking code in prior to starting on the editorconfig portion?  If so, I would wait to add this until after you've checked in this first batch of changes.